### ISSUE TITLE
Production jar includes tests

### ISSUE THREAD

**Issue by mikekap** [ladderlife/om-css#31]
``` bash
$ unzip -l om-css-0.4.10.jar
Archive:  om-css-0.4.10.jar
  Length     Date   Time    Name
 --------    ----   ----    ----
...
        0  03-08-16 22:43   om_css/devcards/
     6172  03-08-16 22:43   om_css/devcards/bugs.cljs
      119  03-03-16 15:52   om_css/devcards/constants.cljc
     4986  03-08-16 22:43   om_css/devcards/core.cljs
      219  02-10-16 23:34   om_css/runner.cljs
    12678  03-08-16 21:23   om_css/tests.clj
     1371  03-08-16 21:46   om_css/tests.cljs
```

**Comment by mikekap** [comment]
I'm told https://github.com/omcljs/om/blob/master/project.clj#L26-L27 is the fix.

**Comment by anmonteiro** [comment]
fixed 6634da8f1da0dc6db850d1464f92d657b0d4a0c2
